apps/opik-backend/pom.xml

```diff
@@ -254,6 +254,12 @@
             <artifactId>mysql</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.clickhouse</groupId>
+            <artifactId>clickhouse-jdbc</artifactId>
```

Comment:
Needed by the New Clickhouse container implementation

Reply:
As discussed, it solves this tech debt item:
- https://github.com/comet-ml/comet-llm-eval-opik/issues/58